sdk/storage/storage-file-share/test/perfstress/track-1/upload.spec.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PerfStressOptionDictionary } from "@azure/test-utils-perfstress";
+import { Aborter, FileURL } from "@azure/storage-file";
+import { StorageFileShareTest } from "./storageTest";
+
+interface StorageFileShareUploadTestOptions {
+  size: number;
+}
+export class StorageFileShareUploadTest extends StorageFileShareTest<
+  StorageFileShareUploadTestOptions
+> {
+  public options: PerfStressOptionDictionary<StorageFileShareUploadTestOptions> = {
+    size: {
+      required: true,
+      description: "Size in bytes",
+      shortName: "sz",
+      longName: "size",
+      defaultValue: 1024
+    }
+  };
+
+  async runAsync(): Promise<void> {
+    const fileClient = FileURL.fromDirectoryURL(
```

Comment:
Each instance should upload to same file

Reply:
Done